### DISCUSSION TITLE
Is it possible to "Go To Definition" on an InlayHint?

### DISCUSSION THREAD

**Discussion by xdesai** [neoclide/coc.nvim#4834] [Q&A]
I've recently started using this plugin and have been loving it. One think I've run into a couple of times though is wanting to jump to the definition on an InlayHint. If I have a hint that tells me an object is of type "first party object" it'd be so great to be able to hover over it and use ctags or something to jump to the definition.

I don't know much about virtual text in vim so this may be impossible, but I was curious to hear if anybody has figured out a way to achieve this.

**Comment by fannheyward** [comment]
Good idea, I think this needs vim/nvim provides a way to put cursor on virtual text.
